tests/test_interface.rb

```diff
@@ -471,7 +476,7 @@ def test_interface_description_valid
     description = 'This is a test description ! '
     interface.description = description
     assert_equal(description.rstrip, interface.description)
-    interface_ethernet_default(interfaces_id[0])
+    interface_ethernet_default(interfaces[0])
```

Comment:
Good point.  I ended up removing almost all of the `interface_ethernet_default` lines from tests except for [this one](https://github.com/cisco/cisco-network-node-utils/pull/300/files#diff-cf2422f8b2cb8aa033a2f17f39d7354fR1359)